cumulus/chain/chaincontext.py

```diff
@@ -5,7 +5,7 @@ class ChainContext:
 
     def __init__(self,
                  template,
-                 instance_name,
+                 instance_name=None,
```

Comment:
This is a deprecation right?

Reply:
Yes.  There is logging now if you put one in it will show.

Reply:
I don't think deprecations should show up as errors.   Lets change it so it's not a big red thing.